app/views/idv/doc_auth/mobile_document_capture.html.erb

```diff
@@ -0,0 +1,109 @@
+<% if Figaro.env.acuant_sdk_document_capture_enabled == 'true' %>
```

Comment:
I'm trying to understand what the future situation will be in consideration of LG-3205 and LG-3251, where at some point we combine these to a single flow. If I understand correctly, a user would arrive at this view if they choose the option to "Use your phone"; immediately if already on a phone, or directed here by a link if they enter a phone number on the next prompt on a desktop computer.

Do you see this screen as being temporary then, until device detection can be done in the browser?

In #3990, I'm starting to explore some of this device-specific logic. It's quite cumbersome to implement, both in how to actually do the device detection, and in how best to reconcile with what Acuant's SDK reports as being a compatible device.

I could see a future scenario where we keep both of these flows managed by the server, and the browser detects device based on the route they're at (`/mobile_document_capture` vs. `/document_capture`). In the end, I think it's a similar result, since server-side we appear to just be doing [the same sort of user-agent sniffing](https://github.com/18F/identity-idp/blob/9dce5e838825b7cd9c4dee2da41d554e8d794cb2/app/controllers/application_controller.rb#L325-L328) that I've [started to implement in #3990](https://github.com/18F/identity-idp/pull/3990/files#diff-c85787f51ddc6d291e1836a4a61c38cbR21-R22). While both are likely imperfect, it'd at least be nice to lean on `DeviceDetector` for consistency.

One of the other highlights is that we're eliminating the user choice to "Use your phone" vs. "Upload", where previously even mobile users would be prompted. This seemed redundant (and from a user's perspective, _is redundant_), but it had some usefulness for us in determining whether we should be prepared to prompt the user to upload, vs. expecting to use Acuant SDK capture.

Reply:
My understanding was that this screen was temporary until we merge them in 3205.